youtube_dl/extractor/archiveorg.py

```diff
@@ -24,12 +24,12 @@ class ArchiveOrgIE(InfoExtractor):
         }
     }, {
         'url': 'https://archive.org/details/Cops1922',
-        'md5': 'bc73c8ab3838b5a8fc6c6651fa7b58ba',
+        'md5': 'bad_md5',
```

Comment:
My bad! I was testing #12855 back then and it seems I've forgotten to save that proper md5...